src/app/lib/utilities/preprocessor/rules/groups.test.js

```diff
@@ -0,0 +1,32 @@
+import { filterEmptyGroupItems } from './groups';
+
+describe('Group rules', () => {
+  it('should filter out empty items', () => {
+    const data = {
+      content: {
+        groups: [
+          {
+            type: 'top-story',
+            items: [{ name: 'item1' }],
+          },
+          {
+            type: 'must-see',
+            items: [],
```

Comment:
Can you please add support for items being undefined, null or a string please. Unlikely to ever happen but good to be safe :)